recipes/netgen/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "netgen" %}
+{% set version = "6.2" %}
+{% set commit = "df71d00d2197a3c69d357cebca3664cf010bd19a" %}
+{% set sha256 = "c86b2b0ecb2b59afebbf50f976ede09dd1bcf05734566a888c9ef24a79d64c76" %}
+
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: netgen{{ version }}.tar.gz
+    url: https://github.com/NGSolve/netgen/archive/{{ commit }}.tar.gz
+    sha256: {{ sha256 }}
+    patches:
+        - python1.patch
+        - salome.patch
```

Comment:
What are these patches for? And have they been sent upstream?

Reply:
The python1.patch includes some conda specific cmake stuff. I have already tried to apply it upstream, but it's a bit difficult. The salome-patch adds some functionality originally added by http://salome-platform.org/. I guess it's better to remove this patch and wait until the PR is merged...